adapters/balance.js

```diff
@@ -52,13 +60,8 @@ module.exports = function (mu, transports) {
     muid: muid,
     tf: tf,
     type: 'transport',
-    setId: setId
-    // driver: driver,
-    // tearDown: tearDown
+    setId: setId,
```

Comment:
Should have removed the commented out line - will do that! Adapters present the same interface as transport.js. i.e. an adapter looks exactly like a transport to the router - make sense ?